src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -67,9 +69,18 @@ struct ONNXConvOpLowering : public ConversionPattern {
 
     // Determine the bounds for the loops over batch & channel out.
     IndexExpr iZero = LiteralIndexExpr(0);
-    ValueRange outerLoops = createKrnl.defineLoops(3);
+    IndexExpr iOne = LiteralIndexExpr(1);
+
+    SmallVector<Value, 3> lbsStorage, ubsStorage, stepsStorage;
     SmallVector<IndexExpr, 3> outerLbs = {iZero, iZero, iZero};
     SmallVector<IndexExpr, 3> outerUbs = {N, G, COPerGroup};
+    SmallVector<IndexExpr, 3> outerSteps = {iOne, iOne, iOne};
+    IndexExpr::getValues(outerLbs, lbsStorage);
+    IndexExpr::getValues(outerUbs, ubsStorage);
+    IndexExpr::getValues(outerSteps, stepsStorage);
```

Comment:
If you want to create directly set of values, and you don't need the indexExpr, you can also use
```C++
SmallVector<Value, 3> lbsStorage = (create.math.constantIndex(0), create.math.constantIndex(0), create.math.constantIndex(0)}
```
or because the 3 vals are identical:
```C++
SmallVector<Value, 3> lbsStorage(3, create.math.constantIndex(0))
```

Reply:
the indexExpr's are also used